packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -155,11 +164,37 @@ const App = (props) => {
         }
     }, [baskets])
 
+    useEffect(() => {
+        const lastLoginTimeStamp = Date.parse(customer.lastLoginTime)
+        const creationTimeStamp = Date.parse(customer.creationDate)
+        const isNewCustomer = lastLoginTimeStamp - creationTimeStamp < NEW_CUSTOMER_MAX_AGE
+        const shouldMerge =
+            customerType === 'registered' &&
+            prevAuthType.current === 'guest' &&
+            !isNewCustomer &&
```

Comment:
Why don't we merge basket for a new customer?

Reply:
New customer is merged automatically from the backend, we don't need to do it on the frontend. If we do, it will return 419 I believe, which is the bug we fixed earlier

Reply:
I feel it would be helpful to add a comment to explain this.

Reply:
Done!